test/integration/targets/filters/runme.sh

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
 
-set -ex
-virtualenv --system-site-packages --python "${ANSIBLE_TEST_PYTHON_INTERPRETER}" "${OUTPUT_DIR}/venv"
-source "${OUTPUT_DIR}/venv/bin/activate"
 set -eux
 
+source virtualenv.sh
```

Comment:
What is the current working directory in this case? I'm concerned about the relative path...

Reply:
The script is sourced from `$PATH`, so the current working directory doesn't matter.

Reply:
Ah, ok